src/ExpenseTrackerGrupo2/Controllers/ExpenseController.cs

```diff
@@ -1,6 +1,138 @@
+using ExpenseTrackerGrupo2.Business.Services.Interfaces;
+using ExpenseTrackerGrupo2.Business.Services.Mappers.Requests.Expenses;
+
+using Microsoft.AspNetCore.Mvc;
+
 namespace ExpenseTrackerGrupo2.Controllers;
 
-public class ExpenseController
+[ApiController]
+[Route("api/expenses")]
+public class ExpenseController : ControllerBase
 {
+    private readonly IExpenseService _expenseService;
+
+    public ExpenseController(IExpenseService expenseService)
+    {
+        _expenseService = expenseService;
+    }
+
+    [HttpPost]
+    public async Task<IActionResult> CreateExpense([FromBody] ExpenseCreateRequest request)
```

Comment:
Post method it works but display a message:
{
  "type": "https://tools.ietf.org/html/rfc9110#section-15.6.1",
  "title": "An error occurred while processing your request.",
  "status": 500
}

yes, its make the Post of the new Expense, but if it create correctly shouldnt show a error message.

Reply:
Done